docs/users-guide/Analyzing-and-Translating-Models.adoc

```diff
@@ -1258,7 +1262,78 @@ and view the layouts in a browser.
 See the https://github.com/fprime-community/fprime-layout[`fprime-layout`
 repository] for more details.
 
-=== Generating JSON
+=== Generating Ground Dictionaries
+
+A *ground dictionary* specifies all the commands,
```

Comment:
I have always heard the term _flight dictionary_, or _flight system dictionary_. In the [spec](https://github.com/nasa/fprime/blob/devel/docs/Design/fpp-json-dict.md#fpp-json-dictionary-specification), we say "FPP JSON dictionaries". Should we consolidate on a single terminology?

Reply:
I vote for calling it the ground dictionary. From the point of view of the FSW framework, it is the dictionary exported to the GDS.